src/systems/ec/implementations/react/templates/pages/search/src/SearchPage.jsx

```diff
@@ -189,145 +189,119 @@ const SearchPage = ({ siteHeader, pageHeader, footer, template }) => (
-            <article className="ecl-u-type-m ecl-u-mt-l ecl-u-mt-lg-m ecl-u-pb-l ecl-u-pb-lg-m">
-              <Link
-                href="/example"
-                label="Justice and fundamental rights"
-                className="ecl-u-type-prolonged-m ecl-u-type-bold"
-              />
-              <p className="ecl-u-type-paragraph-m ecl-u-type-color-grey ecl-u-mv-none">
-                Every EU citizen enjoys the same fundamental rights based on the
-                values of equality, non-discrimination, inclusion, human
-                dignity, freedom and democracy.
-              </p>
-            </article>
-            <hr className="ecl-u-mv-none" />
-            <article className="ecl-u-type-m ecl-u-mt-l ecl-u-mt-lg-m ecl-u-pb-l ecl-u-pb-lg-m">
-              <Link
-                href="/example"
-                label="Migration"
-                className="ecl-u-type-prolonged-m ecl-u-type-bold"
-              />
-              <p className="ecl-u-type-paragraph-m ecl-u-type-color-grey ecl-u-mv-none">
-                The plight of thousands of migrants putting their lives in peril
-                to cross the Mediterranean has shocked. It is clear that no EU
-                country can or should be left alone to address huge migratory
-                pressures.
-              </p>
-            </article>
-            <hr className="ecl-u-mv-none" />
-            <article className="ecl-u-type-m ecl-u-mt-l ecl-u-mt-lg-m ecl-u-pb-l ecl-u-pb-lg-m">
-              <Link
-                href="/example"
-                label="A stronger global actor"
-                className="ecl-u-type-prolonged-m ecl-u-type-bold"
-              />
-              <p className="ecl-u-type-paragraph-m ecl-u-type-color-grey ecl-u-mv-none">
-                The EU needs a strong common foreign policy to respond
-                efficiently to global challenges, including the crises in its
-                neighbourhood, project its values, reject protectionism and keep
-                EU trade standards and contribute to peace and prosperity in the
-                world.
-              </p>
-            </article>
-            <hr className="ecl-u-mv-none" />
-            <article className="ecl-u-type-m ecl-u-mt-l ecl-u-mt-lg-m ecl-u-pb-l ecl-u-pb-lg-m">
-              <Link
-                href="/example"
-                label="Democratic change"
-                className="ecl-u-type-prolonged-m ecl-u-type-bold"
-              />
-              <p className="ecl-u-type-paragraph-m ecl-u-type-color-grey ecl-u-mv-none">
-                For the first time, in 2014, EU countries had to take the
-                results of the elections into account when proposing a candidate
-                for European Commission President.
-              </p>
-            </article>
+
+            <ContentItem
+              title={{
+                href: '/example',
+                label: 'Jobs, growth and investment',
```

Comment:
link is no more underline like current version?

Reply:
well, it is not underlined for all other content items. We could make an exception here, but I think we should be consitent. (though I agree that links should be underlined in theory; it may be updated in content item template directly)